core/controllers/suggestion.py

```diff
@@ -401,7 +455,9 @@ def post(self, suggestion_id):
             )
         )
         _upload_suggestion_images(
-            self.request, updated_suggestion, new_image_filenames)
+            self.payload.get('files', None),
```

Comment:
Can you add a TODO here to refactor this to the new system that you've introduced?

Reply:
Done